src/tiled/brokenlinks.cpp

```diff
@@ -420,6 +477,11 @@ void BrokenLinksWidget::selectionChanged()
             else
                 mLocateButton->setText(tr("Open Tileset..."));
             break;
+        case Tiled::Internal::TemplateGroupReference:
+            if (isTemplateGroup)
+                mLocateButton->setText(tr("Locate File..."));
+            else
+                mLocateButton->setText(tr("Open Template Group..."));
```

Comment:
The button should always read "Locate File..." at this point, since we can't open template groups directly.